itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcrossInstances.java

```diff
@@ -847,21 +848,23 @@ public void testIncrementalDumpMultiIteration() throws Throwable {
             .verifyResults(new String[] {"2" })
             .run("select id from table3")
             .verifyResults(new String[] {"3" });
+    assert(IncrementalLoadTasksBuilder.numIteration > 1);
+    long numIteration = IncrementalLoadTasksBuilder.numIteration;
 
     incremental = primary.run("use " + primaryDbName)
-            .run("create table  acid_table (key int, value int) partitioned by (load_date date) " +
-                    "clustered by(key) into 2 buckets stored as orc tblproperties ('transactional'='true')")
+            .run("create table  table5 (key int, value int) partitioned by (load_date date) " +
+                    "clustered by(key) into 2 buckets stored as orc")
             .run("create table table4 (i int, j int)")
             .run("insert into table4 values (1,2)")
-            .dump(primaryDbName, incremental.lastReplicationId,
-                    Arrays.asList("'hive.repl.dump.include.acid.tables'='true'"));
+            .dump(primaryDbName, incremental.lastReplicationId);
 
     replica.load(replicatedDbName, incremental.dumpLocation, Arrays.asList("'hive.repl.approx.max.load.tasks'='1'"))
             .run("use " + replicatedDbName)
             .run("show tables")
-            .verifyResults(new String[] {"acid_table", "table1", "table2", "table3", "table4" })
+            .verifyResults(new String[] {"table1", "table2", "table3", "table4", "table5" })
             .run("select i from table4")
             .verifyResult("1");
+    assertTrue(IncrementalLoadTasksBuilder.numIteration > 1);
```

Comment:
In this case, number of iterations should be equal to number of events in the dump. Shall do listStatus on loadPath to get events count.

Reply:
done